src/Sulu/Bundle/AdminBundle/Resources/js/views/Form/toolbarActions/CopyToolbarAction.js

```diff
@@ -0,0 +1,89 @@
+// @flow
+import React from 'react';
+import {action, observable} from 'mobx';
+import jexl from 'jexl';
+import log from 'loglevel';
+import Dialog from '../../../components/Dialog';
+import ResourceRequester from '../../../services/ResourceRequester';
+import ResourceStore from '../../../stores/ResourceStore';
+import {translate} from '../../../utils/Translator';
+import {ResourceFormStore} from '../../../containers/Form';
+import Form from '../Form';
+import Router from '../../../services/Router';
+import AbstractFormToolbarAction from './AbstractFormToolbarAction';
+
+export default class CopyToolbarAction extends AbstractFormToolbarAction {
```

Comment:
another possibility would be adding a generic toolbar action (eg. `RequestToolbarAction`) like described in #5342.
such an action would need to support the following configuration parameters:

- `label`
- `icon`
- `visible_condition`
- `disabled_condition`
- `dialog_title`
- `dialog_description`
- `route_name` (target of the request)
- `request_parameters` (additional parameters for the request)
- `router_attributes_to_request` (dynamic parameters for the request)
- `resource_store_properties_to_request` (dynamic parameters for the request)
- `success_view` (target of redirect after success)
- `result_to_view` (attributes for redirect view, similar to deeplinks)

additionally, we would need to make the target of the request configurable. in the `copy` case, we want

Reply:
this would be the better solution - but for now its OK for me